cli/service/service.go

```diff
@@ -105,10 +106,11 @@ func Log(serviceName string, tailing bool) {
 		serviceFound := false
 		var instanceId string
 		for _, serviceInstance := range runningServices.Instances["active"] {
-			if strings.Contains(serviceInstance.SpecRef, refUrl) {
+			org, name := cutil.SplitOrgSpecUrl(refUrl)
+			if (serviceInstance.SpecRef == name && serviceInstance.Org == org) || strings.Contains(serviceInstance.SpecRef, refUrl) {
 				instanceId = serviceInstance.InstanceId
 				serviceFound = true
-				msgPrinter.Printf("Displaying log messages for service %v with service id %v.", serviceInstance.SpecRef, instanceId)
+				msgPrinter.Printf("Displaying log messages for service %v with service id %v.", refUrl, instanceId)
```

Comment:
I think this msgs needs to specify serviceInstance.SpecRef so that the user knows which service we found. refUrl might be a substring of the full service name.

Reply:
Done.